src/main/java/com/google/api/gax/grpc/BundlingCallSettings.java

```diff
@@ -0,0 +1,122 @@
+package com.google.api.gax.grpc;
+
+import com.google.api.gax.core.RetrySettings;
+import com.google.common.collect.ImmutableSet;
+
+import io.grpc.ManagedChannel;
+import io.grpc.MethodDescriptor;
+import io.grpc.Status;
+
+import java.io.IOException;
+import java.util.Set;
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * A settings class with generic typing that could used to configure an bundling method
+ * or create the bundling callable object, which can make API method calls.
+ */
+public class BundlingCallSettings<RequestT, ResponseT>
+    extends ApiCallSettingsTyped<RequestT, ResponseT> {
+  private BundlingDescriptor<RequestT, ResponseT> bundlingDescriptor;
+  private BundlingSettings bundlingSettings;
+
+  ApiCallable<RequestT, ResponseT> create(ServiceApiSettings.Builder serviceSettings)
+      throws IOException {
+    BundlerFactory<RequestT, ResponseT> bundlerFactory = null;
```

Comment:
createFutureCallable() creates the base callable object using the method descriptor.